### ISSUE TITLE
[A11y] Desktop/Mobile Advanced Search: Facets tab items title and number items

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1864]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Advanced Search

### Admin user?

No

### Issue Description

Advanced Search: https://staging.oregondigital.org/advanced

When I tab through the facets and it reads the first and second column items. The screen reader is just reading the title and number.

I ran it through the WAVE tool, its missing fieldsets for the group of checkboxes.

### Suggested Fix

Add fieldsets to the title and item numbers.

Determine whether the grouping of check boxes or radio buttons has or needs text that explains the purpose of the check boxes or radio button grouping. If so, mark up the group within a fieldset and put the group description in a legend element.

### Can anybody QA or does this need an expert?

Anybody

**Comment by jechols** [comment]
@azle: Fieldsets have been implemented, but that doesn't actually address this:

> When I tab through the facets and it reads the first and second column items. The screen reader is just reading the title and number.

This is not an accessibility violation, however, so I'm thinking this can just be marked as "done". The two columns are presented for screen readers in the same way they're presented for sighted users, so it wouldn't really make sense to try and add context that nobody else has. I think this falls into the category of things you just get used to. Sighted users get used to numbers next to these terms meaning the number of items for the given term. Screen reader users will have the same challenge.

Maybe we could file a usability ticket, but I'm almost certain this isn't accessibility.
